pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -935,6 +935,34 @@ public String extractStringValueFrom(ResultSet rs) throws SQLException {
         public String extractStringValueFrom(ResultSet rs) throws SQLException {
           return extractBigDecimalString(rs, getSqlKey());
         }
+      }, //
+      SPIKE_IN("spike_in", "Spike-In"), //
+      SPIKE_IN_DILUTION("spike_in_dilution_factor", "Spike-In Dilution Factor") {
+        @Override
+        public String extractStringValueFrom(ResultSet rs) throws SQLException {
+          String raw = rs.getString(getSqlKey());
+          if (raw == null) {
+            return null;
+          }
+          switch (raw) {
```

Comment:
Maybe these return values should just be integers? I would ask Peter and/or Larry how these get used.

Reply:
:+1: Changed to a decimal after discussing with Peter

Reply:
:+1: Changed to a decimal after discussing with Peter